src/Controls/samples/Controls.Sample.SingleProject/MyApp.cs

```diff
@@ -13,7 +13,14 @@ public class MyApp : MauiApp
 					services.AddTransient<IWindow, MainWindow>();
 				});
 
-		public override IWindow GetWindowFor(IActivationState state) =>
-			Services.GetService<IWindow>();
+		public override IWindow CreateWindowFor(IActivationState state)
+		{
+#if (ANDROID || IOS)
+
+			// This will probably go into a compatibility app or window
+			Microsoft.Maui.Controls.Compatibility.Forms.Init(state);
```

Comment:
Maybe we should do this internally for now? None of the controls can really work without it once we add normal properties. For example, In my Label PR, as soon as I touch the font properties, then it requires that Init be called.

Maybe there is need for overloads and whatnot, so we shouldn't. But just a thought.

But also, maybe we should still call this in the activity and delegate instead of the shared code since this is a compat line?

Reply:
In line with what I mentioned above... I think Init will ultimately go away and all we are going to have left is for users to opt into old school assembly scanning